UnitTests/Core/BorderTests.cs

```diff
@@ -1,19 +1,15 @@
 ﻿using System;
-using System.Reflection.Emit;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
 using Xunit;
-using Xunit.Abstractions;
 using Rune = System.Rune;
 
 namespace Terminal.Gui.CoreTests {
 	public class BorderTests {
-		readonly ITestOutputHelper output;
-
-		public BorderTests (ITestOutputHelper output)
-		{
-			this.output = output;
-		}
-
-		[Fact, AutoInitShutdown]
+		[Fact]
+		[AutoInitShutdown]
```

Comment:
Sorry but I really don't understand what is wrong here. Don't you like them in one line? `Ctrl+K, Ctrl+D` don't modify them, so you spend a lot to change them :-)

Reply:
Was a result of a merge, I think. I didn't do it on purpose.